modules/build/src/main/scala/scala/build/preprocessing/directives/UsingJavaOptionsDirectiveHandler.scala

```diff
@@ -32,14 +33,13 @@ case object UsingJavaOptionsDirectiveHandler extends UsingDirectiveHandler {
 
   override def keys = Seq("javaOpt", "javaOptions", "java-opt", "java-options")
   override def handleValues(
-    values: Seq[Any],
-    cwd: ScopePath,
-    positionOpt: Option[Position]
+    values: Seq[Value[_]],
+    cwd: ScopePath
   ): Either[BuildException, BuildOptions] = {
     val javaOpts = DirectiveUtil.stringValues(values)
     val options = BuildOptions(
       javaOptions = JavaOptions(
-        javaOpts = javaOpts.map(o => scala.build.Positioned(positionOpt.toList, o))
+        javaOpts = javaOpts.map(o => scala.build.Positioned(Seq(o._2), o._1))
```

Comment:
NIT Do a match? like
```scala
javaOpts.map { case (opt, pos) => scala.build.Positioned(Seq(pos), opt) }
```

Reply:
Maybe not necessary if `DirectiveUtil.stringValues` is changed to return `Seq[Positioned[String]]`

Reply:
I've just matched there and left DirectiveUtil as it is